### PR TITLE
Add Elm SPA to skeleton

### PR DESCRIPTION
**Pull request by rl-king**
For now Elm dev is done through Makefiles and template based sites with Gulp. 

There is no further Gulp integration trough Make, so you can basically throw away the gulpfile for template dev and visa-versa for Elm dev. 

- Add makefile
- Add spa tpl
- Update package.json

### CODE REVIEW

File: skel/Makefile [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+	@echo "  build                  Install deps and compile for production"
+	@echo "  clean                  Remove 'dist' folder"
+	@echo "  deps                   Install build dependencies"
+	@echo "  distclean              Remove build files and dependencies"
+	@echo "  help                   Show list of make commands"
+	@echo "  init                   Initialize site, clone Elm and Scss skeletons and install deps"
+	@echo "  minify                 Minify JS and Css files"
+	@echo "  watch                  Compile all Elm and Scss files on file change and reload browser"
+	@echo "  watch-elm              Compile all Elm files on file change and reload browser"
+	@echo "  watch-scss             Compile all Scss files on file change and reload browser"
```
Comment (ddeboer) [inline skel/Makefile]: Some targets, such as `elm`, `assets`, `livereload` and `scss` are missing from this list.

File: skel/config.in [hunk 2]
```diff
@@ -1,5 +1,5 @@
 [
-    {enabled, true},
+    {enabled, false},
```
Comment (ddeboer) [inline skel/config.in]: :+1:

**Comment by ddeboer** [comment]
Conflicts with master.

**Comment by rl-king** [comment]
_Should the skeleton also contain the lib/elm dir, particularly the lib/elm/Makefile?_
Make init takes care of those

File: skel/Makefile (hunk 1 above)
Comment (rl-king) [reply skel/Makefile]: Fixed

**Comment by ddeboer** [comment]
We’ll add the dispatch setup separately; see #408.

**Pull request merged**